### PR TITLE
feat: basic support Apache APISIX 2.13.0

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**
support Apache APISIX 2.13.0

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by guoqqqi** [comment]
@nic-chen Can you guide me through back-end CI issues?

**Comment by guoqqqi** [comment]
@nic-chen Please help me solve the CI error

**Comment by bzp2010** [comment]
@guoqqqi By comparing 2.12 with 2.13 code, I noticed these

- Lack of schema definition for mocking and opentelemetry plugins, we need to check further (We need the full version to compare the two versions)
- We need to add the corresponding options for the new service discovery option `kubernetes`.

cc @nic-chen

File: api/conf/schema.json [hunk 1]
```diff
@@ -401,6 +398,7 @@
 							"OPTIONS",
 							"PATCH",
 							"POST",
+							"PURGE",
```
Comment (bzp2010) [inline api/conf/schema.json:401]: As far as I can see, this change is not included in the APISIX 2.13 release branch, you need to reproduce it and check it.
https://github.com/apache/apisix/blob/release/2.13/apisix/schema_def.lua#L525-L526

File: api/conf/schema.json [hunk 2]
```diff
@@ -1886,6 +1880,35 @@
...
+									"type": "integer"
+								}
+							]
+						}
+					},
+					"required": [
+						"name"
+					],
+					"type": "object"
+				},
```
Comment (bzp2010) [inline api/conf/schema.json:1911]: ditto

**Review by bzp2010 — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Pull request merged**